tools/CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ bin_prefix(oio-file -file-tool)
 target_link_libraries(oio-file
 		oiosds ${GLIB2_LIBRARIES})
 
+add_executable(oio-lb-benchmark oio-lb-benchmark.c)
+bin_prefix(oio-lb-benchmark -lb-benchmark)
+target_link_libraries(oio-lb-benchmark
+		oiocore metautils
+		${GLIB2_LIBRARIES})
```

Comment:
For reviewers: the tool is voluntarily not installed to avoid packaging issues. However we can install it in future 5.x versions.